queries.sql

```diff
@@ -8,3 +8,32 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3 OR (weight_kg = 10.4 OR weight_kg = 17.3);
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] Great job creating a query for the first transaction :clap: However, as requested in the project requirement, it would be nice to verify that the changes were made successfully before and after `ROLLBACK` by adding `SELECT species FROM animals` to the SQL code before and after the `ROLLBACK` keyword as you did in the screenshot